pkg/operator/bootstrap.go

```diff
@@ -145,6 +150,18 @@ func RenderBootstrap(
 	}, {
 		name:     "manifests/machineconfigserver/kube-apiserver-serving-ca-configmap.yaml",
 		filename: "manifests/kube-apiserver-serving-ca-configmap.yaml",
+	}, {
+		name:     "manifests/baremetal/coredns.yaml",
```

Comment:
Is there some way for this to take in platform as a variable:
`/manifests/platform/*`

Reply:
maybe:
`/manifests/infra.Status.APIServerInternalURL/*`

Reply:
`/manifests/spec.Platform/*`